pom.xml

```diff
@@ -31,6 +31,11 @@
             <version>${junit-jupiter.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+            <version>0.8.11</version>
+        </dependency>
```

Comment:
Just wondering, how much test coverage do you have in this task? :)

Reply:
Got the reco for Jacoco using from Sofia, a little bit screwed up, during configuration, but for now, works okay.
After fixing your comments I got 70% from default Intelijj coverage and 91 from Jacoco.
![Uploading Screenshot 2023-10-24 at 10.50.23.png…]()